crates/quill-common/src/events/interact_entity.rs

```diff
@@ -0,0 +1,12 @@
+use feather_ecs::Entity;
```

Comment:
Quill already has an `EntityId` type, so we shouldn't depend on feather-ecs here.

Reply:
Ok